commands/get/template/projects.go

```diff
@@ -0,0 +1,100 @@
+package templates
+
+//handle the project list for rhmap.
+
+import (
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/feedhenry/rhm/commands"
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+//projectsCmd constructs the required writer in order to send the response to the right place.
+type templatesCmd struct {
+	out    io.Writer
+	in     io.Reader
+	getter func(*http.Request) (*http.Response, error)
+	store  storage.Storer
+}
+
+//Project Defines our cli command including its flags and usage then returns the command to allow a user to do specific operations on projects
+func (tc *templatesCmd) Template() cli.Command {
+	return cli.Command{
+		Name:        "projects",
+		Action:      tc.templateAction,
+		Usage:       "project",
+		Description: "projects allows listing projects in rhm",
+	}
+}
+
+//projectAction is where the logic is pulled together to perform the command. This funtion conforms to the cli action
+func (tc *templatesCmd) templateAction(ctx *cli.Context) error {
+	var url = "%sbox/api/templates/projects"
```

Comment:
We could prob make this more generic and take the type in as a argument then do a 

```
var url = "%sbox/api/templates/%s"
fmt.Sprintf(url,host,templateType)
```